src/aap_eda/api/serializers/credential.py

```diff
@@ -78,5 +86,9 @@ class Meta:
             "credential_type",
             "username",
             "vault_identifier",
+            "scm_username",
```

Comment:
@jshimkus-rh can we use username and secret instead of scm_username and scm_password since they serve the same purpose.

Reply:
Can do.